src/main/java/baseball/model/Player.java

```diff
@@ -0,0 +1,51 @@
+package baseball.model;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Player {
+    private String playerNumber;
+
+    public String getPlayerNumber() {
+        return playerNumber;
+    }
+
+    public Player(String playerNumber) {
+        differentToPlayerNumber(playerNumber);
+        isDigitNumber(playerNumber);
+
+        this.playerNumber = playerNumber;
+    }
+
+    // player는 computer와 모든 값이 달라야함
+    public static void differentToPlayerNumber(String str) {
+        if (str.length() != 3) {
+            throw new IllegalArgumentException("입력은 3자리 수");
+        }
+
+        //중복인지 확인
+        Set<Character> set = new HashSet<>();
+        for (int i = 0; i < str.length(); i++) {
+            set.add(str.charAt(i));
+        }
+        if (set.size() != str.length()) {
+            throw new IllegalArgumentException("숫자 중복함");
+        }
+    }
```

Comment:
22 ~ 24 줄과 27 ~ 32줄은 서로 영향을 끼치지 않고 있어요!
다른 메소드로 분리해 어떤 역할을 하는지, 아래처럼 메소드명으로 구분해보는 건 어땠을까요?

```suggestion
    private void validatePlayerNumber(String playerNumber) {
        validateSize(playerNumber);
        validateDuplicates(playerNumber);
    }
```

Reply:
추가로 다른 코드에서 쓸 일이 없어보여, `private` 키워드로 설정했습니다. :)